pt_miniscreen/pages/hud/ap.py

```diff
@@ -19,9 +19,9 @@ def setup_hotspots(self):
         MARGIN_X_LEFT = 30
         MARGIN_X_RIGHT = 10
         SCALE = self.height / 64.0
```

Comment:
Can we avoid duplication of all of these numbers across multiple pages? DRY principle etc.